test/extended/operators/images.go

```diff
@@ -0,0 +1,94 @@
+package operators
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+
+	"github.com/openshift/origin/pkg/oc/cli/admin/release"
+	exutil "github.com/openshift/origin/test/extended/util"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = Describe("[Feature:Platform][Smoke] Managed cluster", func() {
+	f := e2e.NewDefaultFramework("operators")
+
+	It("should ensure pods use images from our release image with proper ImagePullPolicy", func() {
+		// find out the current installed release info
+		var oc = exutil.NewCLI("release-info", exutil.KubeConfigPath())
```

Comment:
this does the same thing as creating default framework - so yo udon't need the default framework.

Reply:
oc wraps a framework basically.